docs/docs/guides/metadata.md

```diff
@@ -0,0 +1,118 @@
+---
+comments: true
+---
+
+# Add metadata to your application
+
+This guide walks you through the usage of context metadata in Keptn.
+
+After reading this guide you will be able to:
+
+- add metadata to applications, workloads
+  and tasks in two ways
+- add metadata information about applications to their traces
+
+## Introduction
+
+Context metadata in Keptn is defined as all additional information that can
+be added to applications, workloads and tasks.
+
+There are two ways to add such metadata: using the `keptn.sh/metadata` annotation and using the  `KeptnAppContext` CRD.
```

Comment:
```suggestion
Context metadata can be added to your application
in either of the following was:

- Use the `keptn.sh/metadata` annotation in your
[KeptnApp](../reference/crd-reference/app.md) resource
- Use the  `KeptnAppContext` resource
```
Sentences beginning with "There are" are inherently weak and should be avoided.  I think putting these options in a list makes them stand out better to people who don't read carefully ;-)

Reply:
this is not 100% the case but I will fix 👍